docs/faq.md

```diff
@@ -9,16 +9,16 @@ need to make a change to your build steps, you are forced to make that change ac
 your similar projects. Using a preset rather than a boilerplate keeps this process DRY.
 
 Tools like [Create React App](https://github.com/facebook/create-react-app) have
-been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
+been fantastic improvements to the tooling ecosystem, but unfortunately, it only works on specific
```

Comment:
Unfortunately "it only works" isn't correct grammar, since the beginning of the sentence starts with "Tools like" which is plural.

This should stay as "only work".

```suggestion
been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
```